api/package.json

```diff
@@ -57,6 +57,7 @@
     "config": "^3.3.3",
     "express": "^4.17.1",
     "faker": "^5.1.0",
+    "fp-ts": "^2.10.5",
```

Comment:
do we still need `purify-ts` in `package.json` and `yarn.lock`?

Reply:
hmm, it was added by mistake while adding the geojson validator, I'll remove it.

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/298/commits/84b4e9453b6c140d631cb3c782904d27ebf2bb9c